opal/mca/threads/pthreads/owner.txt

```diff
@@ -0,0 +1,7 @@
+#
+# owner/status file
+# owner: institution that is responsible for this package
+# status: e.g. active, maintenance, unmaintained
+#
+owner: SNL
+status: unmaintained
```

Comment:
Again, it is odd to add a brand new module and have it set to "unmaintained".

Put differently: there's no reason for the community to accept a brand new module that is defined to be unmaintained.  👎